systemtest/src/test/resources/opa/opa.yaml

```diff
@@ -96,35 +96,35 @@ data:
     deny {
       is_topic_resource
       is_read_operation
-      not consumer_is_whitelisted
+      not consumer_is_allowlisted
```

Comment:
`consumer_is_allowlisted` and `producer_is_allowlisted` seem very weird to me. Maybe you can use `*_is_allowed` or `*_is_on_allowlist`.